specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -613,6 +619,103 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/azureasyncoperations/{operationId}": {
+      "get": {
+        "tags": [
+          "Clusters"
+        ],
+        "operationId": "Clusters_GetAzureAsyncOperationStatus",
+        "description": "The the async operation status.",
+        "x-ms-examples": {
+          "Get Async Operation Status of Creating Cluster": {
+            "$ref": "./examples/GetClusterCreatingAsyncOperationStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./operations.json#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/AsyncOperationResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/updateClusterIdentityCertificate": {
```

Comment:
Is this a newly added API?

Reply:
> Is this a newly added API?

This is an existing API in our server side. S360 detects that this API is called by it hasn't been defined in our swagger spec. Add this API is for fixing completeness issue.

Reply:
> Is this a newly added API?

This is an existing API in our server side. S360 detects that this API is called by it hasn't been defined in our swagger spec. Add this API is for fixing completeness issue.